app-backend/export/src/main/scala/com/azavea/rf/export/model/ExportDefinition.scala

```diff
@@ -0,0 +1,20 @@
+package com.azavea.rf.export.model
+
+import spray.json.DefaultJsonProtocol._
```

Comment:
@notthatbreezy cool! I be glad to. We can do it as a part of unifying ingest and export projects, as ingest project has a similar issue.